docs/v3.x/guides/unit-testing.md

```diff
@@ -0,0 +1,340 @@
+---
+sidebarDepth: 2
+---
+
+# Unit testing
+
+In this guide we will see how you can run basic unit tests for a Strapi application using a testing framework.
+
+::: tip
+In this example we will use [Jest](https://jestjs.io/) Testing Framework with a focus on simplicity and
+[Supertest](https://github.com/visionmedia/supertest) Super-agent driven library for testing node.js HTTP servers using a fluent API
+:::
+
+## Install test tools
+
+`Jest` contains a set of guidelines or rules used for creating and designing test cases - a combination of practices and tools that are designed to help testers test more efficiently.
+
+`Supertest` allows you to test all the `api` routes as they were instances of [http.Server](https://nodejs.org/api/http.html#http_class_http_server)
+
+:::: tabs
+
+::: tab yarn
+`yarn add jest supertest`
+:::
+
+::: tab npm
+`npm install jest supertest`
+:::
+
+Once this is done add this to `package.json` file
+
+add `test` command to `scripts` section
+
+```json
+  "scripts": {
+    "develop": "strapi develop",
+    "start": "strapi start",
+    "build": "strapi build",
+    "strapi": "strapi",
+    "test": "jest --forceExit --detectOpenHandles"
+  },
+```
+
+and add those line at the bottom of file
+
+```json
+  "jest": {
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      ".tmp",
+      ".cache"
+    ],
+    "testEnvironment": "node"
+  }
+```
+
+Those will inform `Jest` not to look for test inside the folder where it shouldn't.
+::::
```

Comment:
This end the tab.

But this should be used just after the `npm` command.

Few lines before.

Reply:
Fixed now